src/tests/sqlsmith/src/sqlreduce/path.rs

```diff
@@ -887,8 +897,29 @@ impl AstNode {
 
             _ => {
                 // Add debug logging for unmatched cases
+                let node_type = match self {
+                    AstNode::Statement(_) => "Statement",
+                    AstNode::Query(_) => "Query",
+                    AstNode::Select(_) => "Select",
+                    AstNode::Expr(_) => "Expr",
+                    AstNode::SelectItem(_) => "SelectItem",
+                    AstNode::TableWithJoins(_) => "TableWithJoins",
+                    AstNode::Join(_) => "Join",
+                    AstNode::TableFactor(_) => "TableFactor",
+                    AstNode::OrderByExpr(_) => "OrderByExpr",
+                    AstNode::With(_) => "With",
+                    AstNode::Cte(_) => "Cte",
+                    AstNode::ExprList(_) => "ExprList",
+                    AstNode::SelectItemList(_) => "SelectItemList",
+                    AstNode::TableList(_) => "TableList",
+                    AstNode::JoinList(_) => "JoinList",
+                    AstNode::OrderByList(_) => "OrderByList",
+                    AstNode::CteList(_) => "CteList",
+                    AstNode::Option(_) => "Option",
```

Comment:
Can just use this instead: https://docs.rs/strum/latest/strum/derive.EnumDiscriminants.html to generate names.

Reply:
We should have some examples inside our codebase.